dashboard/cron.py

```diff
@@ -23,6 +23,16 @@
 
 logger = logging.getLogger(__name__)
 
+# Decorator to clean up function call logging
+def log_function_call(func):
+    @wraps(func)
```

Comment:
I got "name 'wraps' is not defined" error. It is fixed by adding the import line `from functools import wraps`

`Running Crons
========================================
Traceback (most recent call last):
  File "/code/manage.py", line 12, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 442, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.10/site-packages/django/core/management/__init__.py", line 436, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.10/site-packages/django/core/management/base.py", line 412, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.10/site-packages/django/core/management/base.py", line 458, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.10/site-packages/django_cron/management/commands/runcrons.py", line 45, in handle
    crons_to_run = [get_class(x) for x in cron_class_names]
  File "/usr/local/lib/python3.10/site-packages/django_cron/management/commands/runcrons.py", line 45, in <listcomp>
    crons_to_run = [get_class(x) for x in cron_class_names]
  File "/usr/local/lib/python3.10/site-packages/django_cron/__init__.py", line 33, in get_class
    m = __import__(module)
  File "/code/dashboard/cron.py", line 37, in <module>
    class DashboardCronJob(CronJobBase):
  File "/code/dashboard/cron.py", line 159, in DashboardCronJob
    def verify_course_ids(self):
  File "/code/dashboard/cron.py", line 28, in log_function_call
    @wraps(func)
NameError: name 'wraps' is not defined`

Reply:
@jxiao21 Could you please verify whether this is still an issue? Thanks!

Reply:
I added the import and pushed it